share/data-modeling/domain-driven-design/pattern1/README.md

```diff
@@ -0,0 +1,183 @@
+               "type": "CustomerAccount",
+               "property": "customerId"
+           }
+       ]
+   }
+
+```
+The Singleton Value Object ‘Schema Owner’. You can store schema and Uni metadata in it.
+
+```
+"SchemaOwner": {
+           "type": "object",
+           "description": "Schema Owner Details Singleton Value Object",
+           "properties": {
+               "group": {
+                   "type": "string"
+               },
+               "email": {
+                   "type": "string"
+               }
+           }
+       }
+
+```
+
+The ‘CustomerAccount’ entity and its structure.
+
+```
+"CustomerAccount": {
+           "description": "Customer Account",
+           "type": "array",
+           "items": {
+               "type": "object",
+               "properties": {
+		…
+```
+
+
+The nested complex type objects in the ‘CustomerAccount’ such as ‘ContactDetails’.
+
+
+```
+"contactDetails": {
+                       "description": "customer contact details",
+                       "type": "object",
+                       "properties": {
+                           "email": {
+                               "description": "email Id of a customer",
+                               "type": "string"
+                           },
+                           "phone": {
+                               "description": "phone number of a customer",
+                               "type": "string"
+                           }
+                       }
+                   }
+```
+
+The required constraint on the customerId, firstname, lastName, and on the next complex type address, and property of address i.e zipcode.
+
+
+```
+"required": [
+                    "customerId",
+                    "firstName",
+                    "lastName",
+                    "address"
+                ]
+```
+
+4. Press “Create”. Wait for 5 minutes for Share to finish Uni provisioning.
+
+After Uni Provisioning is complete. You will see that the Uni with the name you chose will appear in the running status on the home page. 
+
+# Explore the Uni
+
+Click the Uni you just created. Click on the ‘Entity Explorer’. You will notice ‘Schema Owner’ and CustomerAccount’ Entity’ on the left. 
+
+Schema Owner is a Value Object; It’s a singleton value object with no unique identifier. You will use it to keep track of who is the Uni ‘owner’ and ‘emailId’ of the group owning it. It is good to have this information in the Uni itself. Click Edit and fill the ‘group’ and ‘emailId’ properties.
+
+Let’s create a customer record. Click ‘CustomerAccount’ entity. Click ‘Create Customer Account’. Enter customer account details. For CustomerId, pick a number. I use the [10001,10002…] series for testing. In the real world, either you will have your own sequence generator for customerId, or you will use the customerId from enterprise source of truth systems elsewhere, or you can use the id property value automatically created by Uni for a customer record. 
```

Comment:
Not going to inline edit all of these.  I believe going for `codesnippets` for the items you want to call out by name is a good practice.  Before you make all those changes, @brian-vendia  - does that match our new conventions?